src/main/java/fi/aalto/cs/apluscourses/intellij/actions/CourseProjectAction.java

```diff
@@ -206,11 +206,14 @@ public void actionPerformed(@NotNull AnActionEvent e) {
     Future<Boolean> ideSettingsImported =
         executor.submit(() -> importIdeSettings && tryImportIdeSettings(project, course));
 
+    Future<Boolean> customPropertiesImported =
+        executor.submit(() -> tryImportCustomProperties(project, Paths.get(basePath), course));
```

Comment:
Does this always (re-)download the file even if it was previously downloaded? And the file is only refreshed if the project is "turned into an A+ project" again? Do you know if texts visible in the UI get immediately updated or only after a restart?

Reply:
Now it gets redownloaded every initialization. The text gets updated immediately